samconfig.toml

```diff
@@ -0,0 +1,5 @@
+version = 0.1
+
+[default.deploy.parameters]
```

Comment:
Have we tested this ?

Reply:
yes, I could create the stack using these params
Here is the AWS ref doc https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html